src/components/AppContent/AppContent.vue

```diff
@@ -24,14 +24,34 @@
 
 <template>
 	<main id="app-content-vue" class="app-content no-snapper">
-		<!-- @slot Provide content to the app content -->
-		<slot />
+		<div id="app-content-wrapper">
+			<!-- @slot Provide content to the app content -->
+			<slot />
+			<Splitpanes
+				class="default-theme"
+				@resized="onPaneResize">
+				<Pane class="aside"
+					:size="paneSizes.aside || paneDefaults.aside.size"
+					:min-size="paneDefaults.aside.min"
+					:max-size="paneDefaults.aside.max">
+					<slot name="aside" />
```

Comment:
Please use a different keyword than `aside`, we already have one and it's for the app-sidebar :)

Even if it's not technically an aside per se, I feel it's misleading! :wink:

Reply:
I changed it to `sidebar`, or do you have a less generic suggestion?